provider/datagen/Cargo.toml

```diff
@@ -41,7 +41,7 @@ icu_datetime = { version = "1.2.0", path = "../../components/datetime", features
 icu_decimal = { version = "1.2.0", path = "../../components/decimal", features = ["datagen"] }
 icu_displaynames = { version = "0.10.0", path = "../../experimental/displaynames", features = ["datagen"] }
 icu_list = { version = "1.2.0", path = "../../components/list", features = ["datagen"]}
-icu_locid_transform = { version = "1.2.0", path = "../../components/locid_transform", features = ["datagen"] }
+icu_locid_transform = { version = "1.2.0", path = "../../components/locid_transform", features = ["datagen", "experimental"] }
```

Comment:
Observation: I don't see where `icu_datetime/experimental` gets enabled here.

Reply:
This might not be needed as the data struct is not behind the flag.